components/ai_chat/core/browser/test_utils.cc

```diff
@@ -208,6 +210,44 @@ void ExpectConversationEntryEquals(base::Location location,
   }
 }
 
+void ExpectToolUseEventEquals(base::Location location,
```

Comment:
Might be a bit strange adding this when it's not being used in any tests, but it is related to the new type added in this PR and I need it for 2 other PRs that I can submit independently from each other in the hope of keeping submissions individually small.